### DISCUSSION TITLE
Updating preview images of published mods should not require folder.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4260] [Ideas]
![2023-12-16_21-52-05](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/6d6951d7-6ad7-4895-a56e-29d1d0b1e6f7)
You currently have to specify your mod's folder in order to update the preview icon of the mod. This is a hassle and doesn't really make sense. Details like description, title, and preview icon can be changed easily without actually updating the mod.

*In the past, I had to use the Steam dev console to update my mod previews without changing the version*

**Comment by SDGNelson** [comment]
The game's workshop submission menu does really need improvement. On my to-do list I have a note to add a bunch of QoL features there like saving previous submissions to make it easy to change small details, but I'm not sure when I'll get to it.

**Reply by Jdance-Media** [reply]
👍. Glad to hear that it is on the list.
